sdk/management/pipelines/fluentJavaMgmt.yml

```diff
@@ -84,8 +82,8 @@ jobs:
       - task: Maven@3
         displayName: 'install'
         inputs:
-          mavenPomFile: sdk/management/pom.xml
-          options: '--batch-mode $(DefaultOptions) $(ProfileFlag) -pl !samples javadoc:aggregate -DskipTests=true -DpackageOutputDirectory=$(Build.ArtifactStagingDirectory)" -am'
+          mavenPomFile: pom.xml
+          options: '$(DefaultOptions) $(ProfileFlag) -pl com.azure:azure-parent javadoc:aggregate -DskipTests=true -DpackageOutputDirectory=$(Build.ArtifactStagingDirectory) -Dcheckstyle.skip -am -amd'
```

Comment:
you'll set checkstyle back right

Reply:
Yes, but stylecheck is not in here, is at the last commit I commented.

This checkstyle skip will be always, since the checkstyle tool is incompatible for its checkstyle

Reply:
ok